homeassistant/config_entries.py

```diff
@@ -922,6 +922,29 @@ def _async_in_progress(self) -> List[Dict]:
             if flw["handler"] == self.handler and flw["flow_id"] != self.flow_id
         ]
 
+    @callback
+    def async_create_entry_with_options(
```

Comment:
I'm assuming the reason you didn't call it `async_create_entry` would be W0221 `arguments-differ`?

Reply:
Yup